buildenv/jenkins/common/variables-functions

```diff
@@ -469,6 +477,10 @@ def set_user_credentials() {
     if (!USER_CREDENTIALS_ID) {
          USER_CREDENTIALS_ID = ''
     }
+     GITHUB_API_CREDENTIALS_ID = get_github_user_credentials_id()
+     if (!GITHUB_API_CREDENTIALS_ID) {
+         GITHUB_API_CREDENTIALS_ID = ''
+    }
```

Comment:
This is technically only required in the "pull-request" case at the moment. I was wondering if we should separate it and move it there but I suppose it's not very much overhead to leave it set in all builds. I guess we can leave it.